src/js/webdriver.js

```diff
@@ -112,9 +116,9 @@ gpii.webdriver.navigateHelper = function (that, args) {
  * A helper function to assist in performing a sequence of actions from a single Fluid IoC test sequence step.  See the
  * docs for details.
  *
- * @param that - The component itself
- * @param actions {Object} - An array of action definitions.  Each element is an object with a `functionName` and `args` element.
- * @returns a Promise that will be resolved when the actions are complete, or rejected if there is an error.
+ * @param {Object} that - The component itself
+ * @param {Object} actionDefs - An array of action definitions.  Each element is an object with a `functionName` and `args` element.
```

Comment:
Looks like this is more accurately Array<Something>

Reply:
Good catch.  I went further and added a `@typedef` as well and added other recent improved conventions (mentioning `fluid.promise` in return descriptions, et cetera).